PracticeCode/AntProjects/dividerFunction1.py

```diff
@@ -0,0 +1,13 @@
+#Simple function takes two integers as arguments and divides the first int by the second int.
+#Result will include remainders if any.
+
+
+def divider (number1, number2):
+    # Divide number1 by number2

```

Comment:
Make this a doc above the function name with the following information:
- param name
- accepted type - Integer/Float
- description
- return type

Reply:
See code from python/python for examples